.github/workflows/ci.yml

```diff
@@ -57,3 +57,63 @@ jobs:
         uses: codecov/codecov-action@v1
         with:
           env_vars: OS,PYTHON
+
+  copyright:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Checking that files have copyright headers
+        run: |
+          temp="$(mktemp)"
```

Comment:
This is large enough that it might make sense to have be a separate script; I've been putting these GitHub Action scripts in `scripts/ci/...`; not that it matters that much, and it'll be much better without the node2vec files.

Reply:
good idea, have split out a script in that location